### PR TITLE
ENH [GEAR-274]: Add summary stats

### PR DESCRIPTION
**Pull request by neurosutton**
See issue for background, screenshots of UI output, and testing:
https://flywheelio.atlassian.net/jira/software/projects/GEAR/boards/35?selectedIssue=GEAR-274

Finds MRIQC image quality metric jsons and places the information from the files into the Custom Information tab on the UI.

Passes unit and integration testing.

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+##1.1.7_0.15.2
```
Comment (AndysWorth) [inline docs/release_notes.md:1]: release notes!  Great idea

File: utils/results/store_iqms.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    Locates analysis output. Assumes naming scheme follows BIDS format.
+        output_analysis_id_dir (path): path including the destination id for project
+        level analyses; avoids internal call to get the destination id of the container.
+    Raises:
+        IndexError: If no subject-level jsons are available in the path specified,
+        there is no data that can be harvested for metadata. May need to check the
+        path or see if the analysis was not completed.
+    """
+    try:
+        jsons = glob.glob(op.join(output_analysis_id_dir,'**/*.json'), recursive=True)
```
Comment (AndysWorth) [inline utils/results/store_iqms.py:51]: Nice!

File: utils/fly/dev_helpers.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from os import walk, path
+
+
+def determine_dir_structure(output_analysis_id_dir):
+    """
+    The directory structure during analysis is somewhat obscure.
+    If the file path in the code is incorrect, this method will be called to
```
Comment (AndysWorth) [inline utils/fly/dev_helpers.py:7]: Nice!

File: utils/results/store_bids_tree.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+
+log = logging.getLogger(__name__)
+
+def store_bids_tree(context, run_label, destination_id):
```
Comment (AndysWorth) [inline utils/results/store_bids_tree.py:6]: Not used?  This was stuff for testing what .metadata.json things actually worked.  Did you want to do something with it here?

**Review by AndysWorth — CHANGES_REQUESTED** [review]
This is great!  I really can't find anything constructive to say because everything is done so nicely and completely.

However, through no fault of yours, there is something missing.  Previously, when run at the project level, MRIQC would produce additional output: group statistics and plots.  See "BIDS-App Leves" [here](https://mriqc.readthedocs.io/en/stable/running.html#running-mriqc) and [here](https://mriqc.readthedocs.io/en/stable/reports/group.html).  It looks like the command line is the same as it was so I can only guess that they changed the behavior of MRIQC.  Perhaps the answer is to just use "group" instead of "participant" in the command line.

Excellent work!

**Comment by neurosutton** [comment]
@AndysWorth Added some changes for the tsv's to be available on the Results tab. The final run on the group level shows that, but still has the BIDS validator error (must've forgotten to turn that off). This version was working on the participant level, as well. More comments?

File: utils/results/store_bids_tree.py (hunk 4 above)
Comment (neurosutton) [reply utils/results/store_bids_tree.py:6]: I moved the method into "fly>dev_helpers.py", in case we need it again.

**Comment by neurosutton** [comment]
Unzipped tsv appears on the results tab now, as does mriqc stats on custom information. Going to glance through the code as is myself, but I think a review is worth your time.

File: run.py [hunk 5]
```diff
@@ -270,16 +280,115 @@ def main(gtk_context):
...
+                # Copy the resulting tsv summaries to the enclosing output directory
+                # where the other, zipped output will live.
+                tsvs = glob.glob(os.path.join(output_analysis_id_dir, "*tsv"))
+                for tsv in tsvs:
+                    name_no_tsv = os.path.splitext(os.path.basename(tsv))[0]
+                    dest_tsv = os.path.join(
+                        context.output_dir,
+                        name_no_tsv + "_" + context.destination["id"] + ".tsv",
+                    )
+                    shutil.move(tsv, dest_tsv)
```
Comment (AndysWorth) [inline run.py:324]: This should probably be a copy instead of move so that they are also available in the "complete output" zip archive.

**Review by AndysWorth — CHANGES_REQUESTED** [review]
Looks great!  Sorry for the   one.    last.    change.

File: run.py (hunk 5 above)
Comment (neurosutton) [reply run.py]: Makes sense! Thanks.

**Pull request merged**